scala-package/core/src/main/scala/org/apache/mxnet/NDArray.scala

```diff
@@ -537,6 +551,10 @@ object NDArray {
     new NDArray(handleRef.value)
   }
 
+  private def _crop_assign(kwargs: Map[String, Any] = null)(args: Any*) : NDArrayFuncReturn = {
```

Comment:
why are you defining this function here?

Reply:
Since this function is the only underscore function used for old api, I choose to block the rest underscore function and keep it there.